components/movieNight.py

```diff
@@ -0,0 +1,173 @@
+import discord
+import common.utils as ut
+from components.subcomponents.movieNight import upcomingMovie, eventReminder
+from .subcomponents.movieNight.movie import Movie
+from .subcomponents.movieNight.suggestionDatabase import Suggestions
+
+SUGGESTION_DB_PATH = "./database/movie_suggestion_list.db"
+suggestion_database = Suggestions(SUGGESTION_DB_PATH)
+movie_night_group = discord.app_commands.Group(name="movie", description="Movie night slash commands")
+
+
+# Code that needs to execute every time the bot starts
+def init():
+    upcomingMovie.update_event_description(False)
+    upcomingMovie.start_pick_reminder()
+    eventReminder.start_event_reminder()
+
+
+# add-suggestion command
+# Creates a modal that takes movie name, genre and reason for picking as input
+@movie_night_group.command(name="add-suggestion", description="Suggest a movie using a popup")
+async def add_suggestion(interaction: discord.Interaction):
+    await interaction.response.send_modal(SuggestionModal())
+
+
+# add-suggestion modal
+class SuggestionModal(discord.ui.Modal, title="Suggest a Movie"):
+    movie_name = discord.ui.TextInput(label="Movie Name")
+    movie_genre = discord.ui.TextInput(label="Movie Genre")
+    movie_reason = discord.ui.TextInput(label="Reason for Picking", style=discord.TextStyle.paragraph)
+
+    async def on_submit(self, interaction: discord.interactions):
+        suggested_movie = Movie(self.movie_name.value, self.movie_genre.value, self.movie_reason.value)
+        sender = interaction.user.name
+        reply = suggestion_database.add_suggestion(sender, suggested_movie)
+
+        await interaction.response.send_message(embed=reply, delete_after=ut.DELETE_AFTER_SECONDS)
```

Comment:
Let's not delete these messages. I should document this stuff, but imo we should not delete responses to explicit user commands. We should delete sparingly only to fight against bot spam.

Reply:
Deleting errors should be good too imo.